packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -71,9 +71,12 @@ export class LoginPage {
 
 	/**
 	 * Navigates to the /log-in endpoint.
+	 *
+	 * @param {string} path The path to be appended to /log-in. E.g. /log-in/new is the reskinned login page.
 	 */
-	async visit(): Promise< void > {
-		await this.page.goto( getCalypsoURL( 'log-in' ), { waitUntil: 'networkidle' } );
+	async visit( path = '' ): Promise< void > {
```

Comment:
I'd prefer that this parameter be an optional one instead of using a default value, as it makes things more explicit.

Reply:
```suggestion
	async visit( {path}: {path: string}= {}' ): Promise< void > {
```

Reply:
@worldomonation Not sure if the suggested syntax is correct, I'm getting an error. Could you please check the syntax I've updated in the code and let me know if that is correct?

```
async visit( path?: string ): Promise< void > {
```

Reply:
> @worldomonation Not sure if the suggested syntax is correct, I'm getting an error. Could you please check the syntax I've updated in the code and let me know if that is correct?
> 
> ```
> async visit( path?: string ): Promise< void > {
> ```

I had a stray single quote in the original suggestion, which is why it didn't work.

An optional parameter is better than a default value, but I still think we're better served by explicitly passing in a named parameter. It would end up like this when it is called:

`loginPage.visit( { path: '...' } )`

instead of 

`loginPage.visit( '...' )`

Reply:
Thanks, I've made this change. Could you please check if this looks good to you?